### ISSUE TITLE
Support for ARIA attributes

### ISSUE THREAD

**Issue by JayaKrishnaNamburu** [teleporthq/teleport-code-generators#86]
Can we start adding support for ARIA attributes something like this
~~~
const canHaveAriaLabel = type === 'button' || type === 'input'

  if ((!attrs || !attrs['aria-label']) && canHaveAriaLabel) {
    const label = children ? children.join(' ') : key
    addAttributeToTag(mainTag, 'aria-label', label)
  }
~~~
So main accessible components like ``buttons`` ``input`` ``textarea`` can have a default value if any attribute is not passed from the ``UIDL`` to the generator. I am not sure if it is present in the pipeline, just had an idea of adding support for some basic attributes first.
~~~
     <input aria-label="first-name" />
     <button aria-label="Demo Button" onClick={() => setToggleComponent(true)}>
        Demo Button
      </button>
~~~

**Comment by JayaKrishnaNamburu** [comment]
Will give a small PR, so we can test it out 👍

**Comment by alexnm** [comment]
this can either be done in the main plugins for the component generators, or it can be considered a separate processing step on the UIDL, right after the resolver

**Comment by JayaKrishnaNamburu** [comment]
Any update on how we are going to handle this as we go with the generators or UIDL stage, I am not completely aware of the pros and cons of both approaches.

**Comment by alexnm** [comment]
Nothing yet, it requires additional discussions I think. At the moment we are focused at wrapping the Beta Milestone by the time JSHeroes happens, which is two weeks from now. But this is definitely on the list for after that, as accessibility is an important practice we want to bake into the generators

**Comment by JayaKrishnaNamburu** [comment]
Hey is there a discussion about teleport in JSHero's, awesome, will wait for the live stream if there is any 😅

**Comment by alexnm** [comment]
not really, but we have all the team from Cluj at the event and we hope to engage with attendees on the open source project :)

**Comment by alexnm** [comment]
I would keep this as a reference for now, but we have to come up with a more complex strategy for handling support for a11y

**Comment by alexnm** [comment]
this should be implemented part of #339 before the playground release
